core/modules/field/field.crud.inc

```diff
@@ -226,8 +226,8 @@ function field_update_field($field) {
 /**
  * Validates a field configuration.
  *
- * @param $field_name
- *   The field name to read.
+ * @param $field
+ *   A field structure. $field['field_name'], $field['type'] must be provided.
  * @param $update
  *   Whether this is a new or existing field.
```

Comment:
I know this is what the original comment says, but since the parameter is `$update`, I would say *Whether this is an existing field or a new one.*